crates/red_knot_workspace/tests/check.rs

```diff
@@ -275,7 +275,7 @@ const KNOWN_FAILURES: &[(&str, bool, bool)] = &[
     ("crates/ruff_python_parser/resources/inline/ok/type_param_type_var_tuple.py", true, true),
     ("crates/ruff_python_parser/resources/inline/ok/type_param_type_var.py", true, true),
     ("crates/ruff_python_parser/resources/valid/statement/type.py", true, true),
-    ("crates/ruff_linter/resources/test/fixtures/flake8_type_checking/TCH004_15.py", true, true),
+    ("crates/ruff_linter/resources/test/fixtures/flake8_type_checking/TC004_15.py", true, true),
```

Comment:
Not quite sure why that particular rename was missing from the main branch, I'm pretty sure I got them all originally, since grep didn't return anything. Maybe this was accidentally reverted?